api/generator.py

```diff
@@ -0,0 +1,77 @@
+import face_recognition
+import os
+import cv2
+
+
+KNOWN_FACES_DIR = 'api/known_faces'
+TOLERANCE = 0.6
+FRAME_THICKNESS = 3
+FONT_THICKNESS = 1
+MODEL = 'cnn'
+
+known_faces = []
+known_names = []
```

Comment:
How do you think about getting rid of using global variables here? I suggest to make a function, that will initialize and return this `known_faces` and `known_names`. Then you will invoke this function in api view in `flask_app.py` and pass this variables into `gen()` function.

Reply:
> How do you think about getting rid of using global variables here? I suggest to make a function, that will initialize and return this `known_faces` and `known_names`. Then you will invoke this function in api view in `flask_app.py` and pass this variables into `gen()` function.

Absolutely, you mentioned just these two global variables, what about all others? As I understand it is preferable to get rid of all global variables.